src/ucai/core/client.py

```diff
@@ -43,6 +43,10 @@ def __init__(self, client: Any = None, **kwargs: Any) -> None:  # noqa: ARG002
     def create_function(self, *args: Any, **kwargs: Any) -> Any:
         """Create a function"""
 
+    @abstractmethod
+    def create_python_function(self, *, func: Callable[..., Any], func_comment: str, catalog: str, schema: str) -> Any:
```

Comment:
```suggestion
    def create_python_function(self, *, func: Callable[..., Any], func_comment: str, catalog: str, schema: str) -> None:
```

if this function returns nothing.

Reply:
Updated to return FunctionInfo